src/js/popup.js

```diff
@@ -382,6 +382,11 @@ const Logic = {
       }
     }
   },
+
+  getCurrentPanel() {
```

Comment:
Can we call this `getCurrentPanelElement` to help distinguish it from `_currentPanel` property and the `getPanelSelector` function already defined on `this`?

Reply:
Oh yes, good point! I just changed it! Thank you!